README.md

```diff
@@ -282,6 +282,10 @@ Example valid JWT for the secret `x` (so you can see the structure):
 }
 ```
 
+### Using a different hashing algorithm
```

Comment:
```suggestion
### Using a different signing algorithm
```

Reply:
Some of them are hashing, but some aren't. Collectively they are all signing.

Reply:
Good point, updated.